tools/run_tests/xds_k8s_test_driver/framework/infrastructure/gcp/network_services.py

```diff
@@ -22,9 +23,134 @@
 
 logger = logging.getLogger(__name__)
 
+_ComputeV1 = gcp.compute.ComputeV1
+GcpResource = _ComputeV1.GcpResource
+
```

Comment:
This is the sort of thing that we should encode in our yapf configuration. I imagine introducing this will boil the ocean in the repo, so I've filed [an issue](https://github.com/grpc/grpc/pull/26769#discussion_r678725608) to do this separately.

Reply:
Great point. This should be automated.